firecloud/api.py

```diff
@@ -1155,13 +1155,16 @@ def get_submission_queue():
 ### 1.7 Workspaces
 #####################
 
-def list_workspaces():
+def list_workspaces(fields=None):
     """Request list of FireCloud workspaces.
 
     Swagger:
         https://api.firecloud.org/#!/Workspaces/listWorkspaces
     """
-    return __get("workspaces")
+    if fields == None:
```

Comment:
```suggestion
    if fields is None:
```
OR
```suggestion
    if not fields:
```

Reply:
went with `if fields is None:` to explicitly only match the default arg None as opposed to fields resolving to a boolean